javascript-bank/account.js

```diff
@@ -0,0 +1,50 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if (isNaN(amount)) {
+    return false;
+  } else if (!Number.isInteger(amount)) {
+    return false;
+  } else if (amount <= 0) {
+    return false;
+  } else if (typeof amount === 'string') {
+    return false;
+  } else if (Number.isInteger(amount)) {
```

Comment:
This could just be and `else`, since you're already checking the reverse case on line 11.